scalahost/nsc/src/main/scala/scala/meta/internal/semantic/ConfigOps.scala

```diff
@@ -0,0 +1,12 @@
+package scala.meta.internal
+package semantic
+
+trait ConfigOps { self: DatabaseOps =>
+  object config {
```

Comment:
A counterargument here would be that compiler plugins work differently across different implementation platforms, and we'd like a uniform way to configure semantic dbs regardless of the platform. Wdyt?